code/Model/Recipient/EmailRecipient.php

```diff
@@ -177,7 +177,7 @@ protected function getFormParent()
         }
 
         // Revert to checking for a form from the session
-        // LeftAndMain::sessionNamespace is protected. @todo replace this with a non-deprecated equivalent.
+        // LeftAndMain::sessionNamespace is protected.
         $sessionNamespace = $this->config()->get('session_namespace') ?: CMSMain::class;
```

Comment:
I checked another places where we get `session_namespace` and only here we have notification to `replace this with a non-deprecated equivalent`.